### PR TITLE
Slack integration

### PR DESCRIPTION
**Pull request by lancelafontaine**

### CODE REVIEW

**Comment by jacobrs** [comment]
Can we increase the code coverage on this @lancelafontaine so that we don't have deteriorating test coverage?

**Comment by lancelafontaine** [comment]
@jacobrs Yup. At the moment `pytest` isn't set up at all with ci/coverage, so even if there were tests, this would still fail. That'll be my next task.

File: server/src/controller.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+from config import config
+from sanic.response import json
+import rest_service
+
+def get_heartbeat(request):
+    return json(ok('hello world'))
```
Comment (jacobrs) [inline server/src/controller.py:6]: In the future we can add a bit more information in this heartbeat such as versioning and dependency health (databases...)

File: client/src/components/Login.vue [hunk 2]
```diff
@@ -52,6 +68,8 @@ export default {
         }
       }
       console.log(formValues)
+      // restFactory.login((res) => {})
```
Comment (jacobrs) [inline client/src/components/Login.vue]: will these commented out calls be of use in the immediate future? If not we should remove them.

**Review by jacobrs — APPROVED** [review]
Looks good! I like the restFactory isolation.

File: client/src/components/Login.vue (hunk 2 above)
Comment (lancelafontaine) [reply client/src/components/Login.vue]: Not immediate future, but eventually :). I'll delete them for now.

**Pull request merged**